docker-compose/jaeger-docker-compose.yml

```diff
@@ -3,23 +3,27 @@ version: '2'
 services:
     jaeger-collector:
       image: jaegertracing/jaeger-collector
-      command: ["--cassandra.keyspace=jaeger_v1_dc1", "--cassandra.servers=cassandra", "--collector.zipkin.http-port=9411"]
+      command: ["/bin/wait-for-cassandra.sh", "/go/bin/collector-linux", "--cassandra.keyspace=jaeger_v1_dc1", "--cassandra.servers=cassandra", "--collector.zipkin.http-port=9411"]
```

Comment:
> `"--cassandra.keyspace=jaeger_v1_dc1"

I realize this was there before, but it's a Bad Idea (tm). If this is billed as a reusable docker-compose file, it should be using env vars for such things.

Reply:
Understood. IDK how reusable this compose setup is other than for testing. I guess I understand it could be a basic setup for standalone distribution.